libfwupdplugin/fu-cfi-device.c

```diff
@@ -467,6 +467,64 @@ fu_cfi_device_to_string(FuDevice *device, guint idt, GString *str)
 		fu_common_string_append_kx(str, idt, "BlockSize", priv->block_size);
 }
 
+/**
+ * fu_cfi_device_chip_select:
+ * @self: a #FuCfiDevice
+ * @value: boolean
+ * @error: (nullable): optional return location for an error
+ *
+ * Sets the chip select value.
+ *
+ * Returns: %TRUE on success
+ *
+ * Since: 1.8.0
+ **/
+gboolean
+fu_cfi_device_chip_select(FuCfiDevice *self, gboolean value, GError **error)
```

Comment:
Can we imagine that a programmer support multiple cs pins?

Reply:
Yes, but in that case they'd be multiple FuCfiDevices.